src/Event/ProxyMethodCallEvent.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Event;
+
+final class ProxyMethodCallEvent
+{
+    public function __construct(
+        string $service,
+        string $class,
+        string $method,
+        ?array $arguments,
+        $result,
+        string $status,
+        ?object $error,
+        float $timeStart,
+        float $timeEnd
+    ) {
+        $this->service = $service;
+        $this->class = $class;
+        $this->methodName = $method;
+        $this->arguments = $arguments;
+        $this->result = $result;
+        $this->status = $status;
+        $this->error = $error;
+        $this->timeStart = $timeStart;
+        $this->timeEnd = $timeEnd;
+    }
+
+    public string $service;
```

Comment:
Let's avoid public properties. And please place the properties before `__construct()`

Reply:
This is an event, there is no need to add getters.

Reply:
Why? Public properties is bad practice.

Reply:
> Why? Public properties is bad practice.
I agree.
Events should contain getters when we want to make it immutable.